### ISSUE TITLE
[a11y] radio button lack of focus mgt using tab or shift+tab.

### ISSUE THREAD

**Issue by Etat-Plateforme** [angular-ui/bootstrap#4251]
According to http://www.w3.org/TR/wai-aria-practices/#radiobutton the focus management using tab or shift+tab should be:
* _tab_ key will enter the radio group:
  * When Tab or Shift+Tab into a radio group, focus goes to the selected radio button. If none is selected, focus goes to the first radio button if Tab was pressed, or the last radio button if Shift+Tab was pressed.
  * When focus is on any radio button, Tab or Shift+Tab will exit the radio group.

* _up arrow_ and _left arrow_ moves focus to the previous radio button in the group, and selects that button. If focus is on the first item, then focus wraps to last item.
* _down arrow_ and _right arrow_ moves focus to the next radio button in the group, and selects that button. If focus is on the last item, then focus wraps to first item.

**Comment by wesleycho** [comment]
For the tab/shift+tab behavior, I am not sure there is anything we can do - however, the user can make use of `aria-activedescendant` to accomplish this in the button group container.

I think we may be able to implement the latter two bullets, but that requires some thought/implemenation work.

**Comment by wesleycho** [comment]
Another alternative approach is for us to create an optional button group directive with registration controller where each button can be registered, and have that handle focus management.

**Comment by icfantv** [comment]
Closing in favor of #4772.
